pkgs/applications/editors/vscode/extensions/default.nix

```diff
@@ -24,6 +24,7 @@
 , typst-lsp
 , typst-preview
 , autoPatchelfHook
+, xsel
```

Comment:
If this extension needs extra parameters it is ideal to keep it in a separate directory. Because it can grow, and in case the extension is removed no one remembers to remove these parameters from the global index.

Reply:
My understanding was that separate separate files were used for complex additions (lots of additional dependencies, complex patching logic, etc). Which this package used to have (it bundled a native service, which required quite a bit of support from us to make work), but no longer needs. At this point it's on par with several other extensions (`azdavis.millet`, `b4dm4n.vscode-nixpkgs-fmt`, `mgt19937.typst-preview`, etc.).

I've added a note to the import to more easily see why it's being used; is this a reasonable compromise?